common/predictive-text/lmlayer-interface.ts

```diff
@@ -0,0 +1,143 @@
+///<reference path="workerFactory.ts" />
```

Comment:
I'm confused by the naming of this file. It's exports an abstract class and not an interface? Why not call it `lmlayer-base` if that's what it contains?